inst/vpa.m

```diff
@@ -62,8 +62,15 @@
         'x, n = _ins'
         'return sympy.N(x, n),' };
     r = python_cmd (cmd, x, n);
-  elseif (ischar(x))
-    x = magic_str_str(x);
+  elseif (ischar (x))
+    if (strcmp (x, 'inf') || strcmp (x, 'Inf') || strcmp (x, '+inf') || ...
```

Comment:
Wow, cool, I will think about this!  Did you mean `x = sym(x)`?  Otherwise I don't see how that could work.